bot/cogs/snakes.py

```diff
@@ -40,8 +65,84 @@ def __init__(self, bot: AutoShardedBot):
         :param ctx: Context object passed from discord.py
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
+        # Everything with snek_list should be cached >(PC)
+        # SELF.BOT.SNEK_LIST OMG OMG OMG >(PC)
+        # Since, on restart, the bot will forget this, it will be re-cached every time. Problem Solved in theory. >(PC)
+        possible_names = 'https://en.wikipedia.org/w/api.php?action=query&titles=List_of_snakes_by_common_name' \
+                         '&prop=extracts&exlimit=1&explaintext&format=json&formatversion=2'
+
+        async with ClientSession() as session:
+            async with session.get(possible_names) as all_sneks:
+                resp = str(await all_sneks.read(), 'utf-8')
```

Comment:
What's the reason for specifying the encoding?

Reply:
There was a problem with encoding when trying to change the read response to a string. Specification solved that.